pkg/roachprod/roachprod.go

```diff
@@ -1349,9 +1371,11 @@ func InitGrafana(
 	if err != nil {
 		return err
 	}
-
+	// Configure the prometheus/grafana servers to run on the last node in the cluster
 	var promCfg prometheus.Config
-	promCfg.WithPrometheusNode(nodes[0])
+	promCfg.WithPrometheusNode(nodes[len(nodes)-1])
```

Comment:
See comment above for why the use of TargetNodes is buggy in this case.

Reply:
Done.